src/Utils.js

```diff
@@ -1,3 +1,78 @@
 (function(){
   LifeGamification.utils = {};
+  LifeGamification.work = {};
+  LifeGamification.refreshTimer;
+
+  LifeGamification.utils.setStartTime = function () {
+    return new Promise((resolve, reject) => {
+      LifeGamification.repository.getWork()
+      .then(function(result){
+        if(result){
+          LifeGamification.work = result;
+          resolve();
+        }
+      });
+    });
+  }
+
+  LifeGamification.utils.startTiming = function () {
+    if(!($('.timer__select-skill').val())){
+      console.log("Error: Add a skill first.");
+      return;
+    }
+    let currentTime = new Date().getTime();
+    let skillName = $('.timer__select-skill').val();
+    LifeGamification.repository
+      .setWork({startTime: currentTime, name: skillName})
+    .then(LifeGamification.utils.handleTimer());
+  }
+
+  LifeGamification.utils.endTiming = function () {
+    const skill = LifeGamification.
+      skillsCollection[LifeGamification.work.name];
+    const expToAdd = Math.floor(calcTime() / 60);
+    LifeGamification.models.updateExp(skill, expToAdd);
+
+    LifeGamification.repository.setWork({});
+    clearInterval(LifeGamification.refreshTimer);
+    LifeGamification.view.finishTimer();
+  }
+
+  displayTime = function(time) {
+    let hours = Math.floor(time / 3600);
+    time %= 3600;
+    let minutes = Math.floor(time / 60);
+    time %= 60;
+    let seconds = time;
+    if(hours < 10){
+      hours = "0" + hours;
+    }
+    if(minutes < 10){
+      minutes = "0" + minutes;
+    }
+    if(seconds < 10){
+      seconds = "0" + seconds;
+    }
+    return `${hours}:${minutes}:${seconds}`;
+  }
+
+  calcTime = function () {
+    let currentTime = new Date().getTime();
+    let timeDiff = Math.floor(
+      (currentTime - LifeGamification.work.startTime) / 1000);
```

Comment:
This code is unextendable. It's depends of LifeGamification.work object but it could not to. Even if you got global variables like this one you should pass the variable via function argument.